nf_core/version.py

```diff
@@ -0,0 +1 @@
+__version__ = "2.4dev"
```

Comment:
Any reason for this change? Should probably update the release checklist in .github too..

Reply:
Yes. I tried to comment shortly on the commit. With this you can debug the tool without installing/building it. Or was that possible before and it missed it?
I also had issues with choosing the correct python installation on gitpod so this might have overlapped.

Reply:
I just tested and I think it works as expected. So it was the wrong python path. I will revert that part

Reply:
Ah ok! Yeah I don't really mind - I've seen various ways of defining the version and was mostly curious if one was more Pythonic than another or if this change fixed some issue that I wasn't aware of.

Reply:
I thought maybe the pkg introspection required an install first. But I think just the package for the import was missing in my python.